albam/engines/mtfw/mesh.py

```diff
@@ -1465,29 +1495,22 @@ def _export_vertices(app_id, bl_mesh, mesh, mesh_bone_palette, dst_mod, bbox_dat
                 if MAX_BONES == 2:
```

Comment:
This `ifs` are getting out of hand. Using [pattern matching](https://peps.python.org/pep-0636/_ in Python might be cleaner at this point.
But probably not in this PR.